entrypoint.sh

```diff
@@ -30,7 +30,12 @@ then
     # Switch to branch from current Workflow run
     git checkout $INPUT_BRANCH
 
-    git add .
+    if [ -z ${INPUT_FILE_PATTERN+x} ];
```

Comment:
Wouldn't it be suffice to simply do `git add $INPUT_FILE_PATTERN` instead of the if-else case as the input param already defaults to `.`?

Reply:
In my test the `$INPUT_FILE_PATTERN` wasn't always set. Maybe that's related with the missing `required`-field in the `actions.yml`? 

I also would like to keep the code as simple as possible, will test the Action with the updated `actions.yml`

Reply:
> Maybe that's related with the missing required-field in the actions.yml?

The doc doesn't mention what happens if the field is omitted, but it's most likely to be the root cause of the issue.

Reply:
Unfortunately this didn't work. The action still fails: `/entrypoint.sh: line 40: INPUT_FILE_PATTERN: parameter not set`.

Reply:
That's interesting. I'll be able to take a deeper look into it later tomorrow.

Reply:
Feel free to tinker with the code. 
I will merge this PR and tag a new version now, as it already works fine.